app/assets/javascripts/discourse/tests/integration/components/admin-schema-theme-setting/editor-test.gjs

```diff
@@ -651,14 +651,20 @@ module(
               default: "awesome",
               choices: ["nice", "cool", "awesome"],
             },
+            required_enum_field: {
+              type: "enum",
+              default: "awesome",
+              required: true,
+              choices: ["nice", "cool", "awesome"],
```

Comment:
I have a silly question, is there a chance that the property the user fills in will not match? e.g. default doesn't exist in choices?

Reply:
There is that possibility but we currently don't verify this when they upload the theme. I'm not too worried about this because this is what site setting does as well and we haven't really ran into issues where the developer filled in the wrong default. Also, a wrong default will lead to very obvious errors when trying to save the theme.